anime_downloader/players/mpv.py

```diff
@@ -19,7 +19,12 @@ def _get_executable_posix(self):
 
     @property
     def args(self):
-        return ['--input-conf='+get_mpv_configfile(), self.stream_url]
+        _args = []
+        _args.append('--input-conf='+get_mpv_configfile())
+        if self.episode.sitename == "twist.moe":
```

Comment:
Can you change this to use `self.episode.source().referer`?

With the recent update I made, this should work(?).

Reply:
I can confirm that it does work with this change made. Passing an episode instead of a URL breaks one of the tests for mpv though. Fixing it would either involve creating an actual anime and episode with one of the sites or implementing some sort of mock objects. The first method would be more tightly coupled than is ideal for a unit test. I took a stab at modifying the base classes to allow them to be instantiated without an actual site but was unsuccessful.

Reply:
Comment out the test for now and open an issue. We can take care of that later.

Reply:
Will do.

Reply:
Created issue #303.